### ISSUE TITLE
Error searching in RELATIONAL FIELDS

### ISSUE THREAD

**Issue by cabohe** [funkjedi/acf-qtranslate#64]
When i setup a relational field ( Page Link / Post Object / Relationship / Taxonomy) , i get an error searching posts. 

If i deactivate your plugin and let active ACF and qTranslate all works fine. :(

![acf-translate-error](https://cloud.githubusercontent.com/assets/12593995/15047648/20b3b5ae-12e6-11e6-9e53-939d10b4af6d.png)

**Comment by fburatti** [comment]
Unfortunately acf-qtranslate does not support relational field, but only Text, Text Area, WYSIWYG, Image and File.

**Comment by fburatti** [comment]
working solution with ACF PRO 5.3.7 (relationship, post object and page link fields):
```php
function my_relationship_result( $title, $post, $field, $post_id ) {

    $title =  qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage($title);
	
    return $title;

}

add_filter('acf/fields/relationship/result', 'my_relationship_result', 10, 4);
add_filter('acf/fields/post_object/result', 'my_relationship_result', 10, 4);
add_filter('acf/fields/page_link/result', 'my_relationship_result', 10, 4);
```

**Comment by cabohe** [comment]
Thanks! With this code and your next report / fix (#65) all works fine!

Good Job!

**Comment by dsentker** [comment]
I like your solution, @fburatti , but it does not use the current edit language (it uses the current admin language, which is not what an editor expects).

Here is a modified version of the code, but it recognize the last setted edit language.

```php
function my_relationship_result( $title, $post, $field, $post_id ) {

    $editLang = $_COOKIE['qtrans_edit_language'];
    if($editLang) {
        return qtranxf_use($editLang, $title, false, false);
    } else {
        return  qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage($title);
    }
}
add_filter('acf/fields/relationship/result', 'my_relationship_result', 10, 4);
add_filter('acf/fields/post_object/result', 'my_relationship_result', 10, 4);
add_filter('acf/fields/page_link/result', 'my_relationship_result', 10, 4);
```
